allvers/2005-05-11-GPL-2.0-from-fsf.org.txt.notes

```diff
@@ -0,0 +1,3 @@
+Link: http://web.archive.org/web/20050511030123/http://www.fsf.org:80/licensing/licenses/gpl.txt
+Date: 20050511
```

Comment:
Although likely this day I could only reliably find a range May 1 - 11.

Reply:
Updated in 2effc1a01fdd1bd1f2486bb2240f78ee65761235